fearless/Common/Substrate/UsedRuntimeModules.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+struct UsedRuntimePaths {
+    private let storageCodingCases = StorageCodingPath.allCases
+    private let callCodingCases = CallCodingPath.allCases
+    private let constantsCodingCases = ConstantCodingPath.allCases
+    private let sabstrateCodingCases = SubstrateCallPath.allCases
+
+    lazy var usedRuntimePaths: [String: [String]] = {
```

Comment:
looks not optimzed. can you think about use compactMap and reduce along with Set, instead of 'forEach'. It should works faster and looks better

Reply:
forEach is faster than reduce, compactMap is not beautiful for this case
https://www.skoumal.com/en/performance-of-built-in-higher-order-functions-map-filter-reduce-and-flatmap-vs-for-in-loop-in-swift/